apps/quickbooks_online/utils.py

```
@@ -1411,3 +1411,32 @@ def get_or_create_entity(self, expense_group: ExpenseGroup, general_settings: Wo
+    def get_accounting_fields(self, resource_type: str):
+        """
+        Retrieve accounting fields for a specific resource type and internal ID.
+
+        Args:
+            resource_type (str): The type of resource to fetch.
+
+        Returns:
+            list or dict: Parsed JSON representation of the resource data.
+        """
+        module = getattr(self.connection, resource_type)
+        generator = getattr(module, 'get_all_generator')()
```

Ruff (0.7.0)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)